packages/contracts/contracts/minter-suite/Minters/MinterSetPriceV5Holder.sol

```diff
@@ -0,0 +1,795 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
nit: I think these "extension" minters should probably be named `MinterSetPriceHolderV5` (or whatever version number is appropriate) to enable us to roll out a minter with new holder logic without rolling the version of MinterSetPrice. Thoughts?

Reply:
Yes i agree 👍 👍  (kinda looks funky to not have the version be the last bit) will update